backend/benefit/messages/views.py

```diff
@@ -49,7 +48,12 @@ class HandlerMessageViewSet(ApplicantMessageViewSet):
 
     def perform_create(self, serializer):
         message = serializer.save()
-        send_email_to_applicant(message.application)
+        # Never send email if it's a handler's note!
+        if (
+            message.message_type == MessageType.HANDLER_MESSAGE
```

Comment:
Not sure about the logic here. If `MessageType.APPLICANT_MESSAGE` evaluates as true, this comparison always evaluates as true, despite the contents of `message`'s `message_type` attribute.

I believe that the intention here was to compare `message_type` to both `MessageType.HANDLER_MESSAGE` and `MessageType.APPLICANT_MESSAGE` which would be in python:
```python
if message.message_type == MessageType.HANDLER_MESSAGE or message.message_type == MessageType.APPLICANT_MESSAGE:
```
This is a bit too verbose, so it could be shorthanded using `list`'s `in` operator:
```python
if message.message_type in [MessageType.HANDLER_MESSAGE, MessageType.APPLICANT_MESSAGE]:
```